lib/spoom/cli/srb/sigs.rb

```diff
@@ -65,6 +51,28 @@ def collect_files(paths)
 
             files
           end
+
+          def transform_files(files, &block)
+            transformed_count = 0
+
+            files.each do |file|
+              contents = File.read(file)
+
+              if contents.lines.first&.start_with?("# encoding:")
+                encoding = T.must(contents.lines.first).gsub(/^#\s*encoding:\s*/, "").strip
```

Comment:
Since we do this for each file it might be worth perf wise to store `contents.lines.first` in a variable and above and reuse it below.

Reply:
I'm confused, can you explain what you mean?